uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Helpers/SessionHelper.php

```diff
@@ -1,18 +1,18 @@
 <?php
 
-trait CRM_HRCore_Test_Helpers_SessionHelpersTrait {
+class CRM_HRCore_Test_Helpers_SessionHelper {
```

Comment:
Does it make sense for the HREmergency extension to have its own session helpers trait rather than installing an extension just to use the session helper methods ?
If No, then probably extensions like L&A and other extension that use their own session helpers trait should later eventually rather use the one from HRCore instead of having the same traits repeated all around

Reply:
Besides the SessionHelper, we also use the ContactFabricator from hrcore (and we can have more in the future!). I don't think it makes sense to duplicate these two classes just to avoid installing hrcore.

You're a right about the other extensions. Eventually, they will have to be updated later to use the hrcore helper instead of they own implementation.